test/utils/format-versions.spec.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+const formatVersions = require("../../utils/format-versions");
+
+describe("#formatVersions", () => {
+  context("when package are present", () => {
+    it("should return a handlebar compile template", () => {
+      const data = {
+        packages: {
```

Comment:
This may be too robust but I want to make sure it hit the `pkgNamePath` function.